mwe_unify.py

```diff
@@ -1,14 +1,22 @@
+test_add_unify_client
+import logging
+import os
+from dotenv import load_dotenv
+
 import dsp
 import dspy
 from dspy.datasets.gsm8k import GSM8K, gsm8k_metric
 from dspy.evaluate import Evaluate
 from dspy.teleprompt import BootstrapFewShot
 
-endpoint = dsp.Unify(
+
+load_dotenv()
+unify_api_key = os.getenv("UNIFY_KEY")
+
+lm = dsp.Unify(
```

Comment:
Hello. In my opinion "lm" must be "endpoint" to configure dspy, I think dspy.settings.configure is expecting an Unify class object